timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -282,6 +285,30 @@ public void multipleLockRequestsWithTheSameIdAreGranted() {
         }
     }
 
+    @Test
+    @Ignore // TODO (jkong): Fix this test by reworking the threading model.
+    public void stressTest() {
+        TestableTimelockServer nonLeader = Iterables.getFirst(cluster.nonLeaders(client.namespace()).values(), null);
+        int startingNumThreads = ManagementFactory.getThreadMXBean().getThreadCount();
+        try {
+            for (int i = 0; i < 10_000; i++) { // Needed as it takes a while for the thread buildup to occur
+                client.getFreshTimestamp();
+                assertThat(ManagementFactory.getThreadMXBean().getThreadCount())
```

Comment:
This test will fail even if the buildup occurs in the warmup period. Maybe at least indicate in the reason for failure if it occurred during warmup or after we simulated shutting down?

Reply:
Alright, will do. Yeah, that seems unlikely, but good to catch :)